base_move:
  pos_tolerance: 0.02
  ang_tolerance: 0.006
  linear_gain: 0.4
  angular_gain: 1.0